--- gst-plugins-bad-0.10.23/configure.ac~	2013-11-03 20:41:07.000000000 +0100
+++ gst-plugins-bad-0.10.23/configure.ac	2013-11-03 20:53:34.929518067 +0100
@@ -1336,7 +1336,7 @@
 dnl *** neon ***
 translit(dnm, m, l) AM_CONDITIONAL(USE_NEON, true)
 AG_GST_CHECK_FEATURE(NEON, [neon http client plugins], neonhttpsrc, [
-  PKG_CHECK_MODULES(NEON, neon >= 0.27.0  neon <= 0.29.99, HAVE_NEON="yes", [
+  PKG_CHECK_MODULES(NEON, neon >= 0.27.0  neon <= 0.39.99, HAVE_NEON="yes", [
     HAVE_NEON="no"
   ])
   AC_SUBST(NEON_CFLAGS)
