diff -urNp -x '*.orig' gst-plugins-bad-0.10.23.org/configure.ac gst-plugins-bad-0.10.23/configure.ac
--- gst-plugins-bad-0.10.23.org/configure.ac	2021-10-06 23:27:49.797106808 +0200
+++ gst-plugins-bad-0.10.23/configure.ac	2021-10-06 23:27:50.470437101 +0200
@@ -840,14 +840,14 @@ dnl *** DIVX ***
 translit(dnm, m, l) AM_CONDITIONAL(USE_DIVX, true)
 AG_GST_CHECK_FEATURE(DIVX, [divx plugins], divx, [
   HAVE_DIVX=yes
-  AC_CHECK_HEADER(encore2.h, , [
+  AC_CHECK_HEADER(divx/encore2.h, , [
     AC_MSG_WARN([Divx4linux encore headers not found])
     HAVE_DIVX=no
   ])
   if [ test x$HAVE_DIVX = xyes ]; then
     AC_MSG_CHECKING([Checking for valid divx4linux encore version])
     AC_TRY_COMPILE([
-#include <encore2.h>
+#include <divx/encore2.h>
 #if ENCORE_VERSION != 20021024
 #error Wrong version of divx encore libraries
 #endif
@@ -861,7 +861,7 @@ return 0;
     ])
   fi
   if [ test x$HAVE_DIVX = xyes ]; then
-    AC_CHECK_HEADER(decore.h, , [
+    AC_CHECK_HEADER(divx/decore.h, , [
       AC_MSG_WARN([Divx4linux decoder headers not found])
       HAVE_DIVX=no
     ])
@@ -869,7 +869,7 @@ return 0;
   if [ test x$HAVE_DIVX = xyes ]; then
     AC_MSG_CHECKING([Checking for valid divx4linux decore version])
     AC_TRY_COMPILE([
-#include <decore.h>
+#include <divx/decore.h>
 #if DECORE_VERSION != 20021112
 #error Wrong version of divx decore libraries
 #endif
diff -urNp -x '*.orig' gst-plugins-bad-0.10.23.org/ext/divx/gstdivxdec.h gst-plugins-bad-0.10.23/ext/divx/gstdivxdec.h
--- gst-plugins-bad-0.10.23.org/ext/divx/gstdivxdec.h	2011-12-11 19:49:11.000000000 +0100
+++ gst-plugins-bad-0.10.23/ext/divx/gstdivxdec.h	2021-10-06 23:27:50.470437101 +0200
@@ -21,7 +21,7 @@
 #define __GST_DIVXDEC_H__
 
 #include <gst/gst.h>
-#include <decore.h>
+#include <divx/decore.h>
 
 #ifdef __cplusplus
 extern "C" {
diff -urNp -x '*.orig' gst-plugins-bad-0.10.23.org/ext/divx/gstdivxenc.c gst-plugins-bad-0.10.23/ext/divx/gstdivxenc.c
--- gst-plugins-bad-0.10.23.org/ext/divx/gstdivxenc.c	2011-12-30 19:24:28.000000000 +0100
+++ gst-plugins-bad-0.10.23/ext/divx/gstdivxenc.c	2021-10-06 23:27:50.470437101 +0200
@@ -24,7 +24,7 @@
 #include <string.h>
 #include "gstdivxenc.h"
 #include <gst/video/video.h>
-#include <encore2.h>
+#include <divx/encore2.h>
 
 static GstStaticPadTemplate sink_template = GST_STATIC_PAD_TEMPLATE ("sink",
     GST_PAD_SINK,
