--- gst-plugins-bad-0.10.23/configure.ac.orig	2012-10-06 08:00:31.847697403 +0200
+++ gst-plugins-bad-0.10.23/configure.ac	2012-10-06 08:29:02.444328416 +0200
@@ -1372,7 +1372,7 @@
   dnl a new version and the no-backward-compatibility define. (There doesn't
   dnl seem to be a switch to suppress the warnings the cvcompat.h header
   dnl causes.)
-  PKG_CHECK_MODULES(OPENCV, opencv >= 2.0.0 opencv <= 2.3.1 , [
+  PKG_CHECK_MODULES(OPENCV, opencv >= 2.0.0 opencv < 2.5.0 , [
     AC_PROG_CXX
     AC_LANG_CPLUSPLUS
     OLD_CPPFLAGS=$CPPFLAGS
