--- gst-plugins-bad-0.10.6/configure.ac~	2008-02-21 15:24:00.000000000 +0100
+++ gst-plugins-bad-0.10.6/configure.ac	2008-02-21 15:55:33.786497575 +0100
@@ -772,16 +772,7 @@
   PKG_CHECK_MODULES(TIMIDITY, libtimidity, [
     HAVE_TIMIDITY="yes",
 	AC_MSG_CHECKING([for timidity.cfg])
-	timidity_cfg=""
-	if test -r /etc/timidity.cfg; then
-	  timidity_cfg=/etc/timidity.cfg
-	elif test -r /etc/timidity/timidity.cfg; then
-	  timidity_cfg=/etc/timidity/timidity.cfg
-	elif test -r /usr/share/timidity/timidity.cfg; then
-	  timidity_cfg=/usr/share/timidity/timidity.cfg
-	elif test -r /usr/local/share/timidity/timidity.cfg; then
-	  timidity_cfg=/usr/local/share/timidity/timidity.cfg
-	fi
+	timidity_cfg="/etc/timidity.cfg"
 	if test "x$timidity_cfg" != "x"; then
 	  AC_MSG_RESULT($timidity_cfg)
       AC_DEFINE_UNQUOTED(TIMIDITY_CFG, "$timidity_cfg", [Define location of timidity.cfg])
--- gst-plugins-bad-0.10.23/ext/timidity/gsttimidity.c.orig	2011-12-30 19:24:28.000000000 +0100
+++ gst-plugins-bad-0.10.23/ext/timidity/gsttimidity.c	2017-12-20 21:07:07.223418598 +0100
@@ -405,7 +405,7 @@
       gst_segment_set_seek (timidity->o_segment, rate, dst_format, flags,
           start_type, start, stop_type, stop, &update);
 
-      if ((flags && GST_SEEK_FLAG_SEGMENT) == GST_SEEK_FLAG_SEGMENT) {
+      if ((flags & GST_SEEK_FLAG_SEGMENT) == GST_SEEK_FLAG_SEGMENT) {
         GST_DEBUG_OBJECT (timidity, "received segment seek %d, %d",
             (gint) start_type, (gint) stop_type);
       } else {
@@ -615,7 +615,7 @@
     GST_DEBUG_OBJECT (timidity, "Parsing song");
 
     stream =
-        mid_istream_open_mem (timidity->mididata, timidity->mididata_size, 0);
+        mid_istream_open_mem (timidity->mididata, timidity->mididata_size);
 
     timidity->song = mid_song_load (stream, timidity->song_options);
     mid_istream_close (stream);
